docs/develop/debug-office-add-ins-in-visual-studio.md

```diff
@@ -68,89 +60,133 @@ The following table describes the properties of the web application project that
+
+4. Opens the Office application and sideloads your add-in.
+
+### Debug the Excel, PowerPoint, or Word add-in
+
+1. Launch the add-in in the Office application. For example, if it is a task pane add-in, it will have added a button to the **Home** ribbon (for example, a **Show Taskpane** button). Select the button in the ribbon. 
+
+   > [!NOTE]
+   > If your add-in is not sideloaded by Visual Studio, you can sideload it manually. In Excel, PowerPoint, or Word, choose the **Insert** tab and then choose the down-arrow located to the right of **My Add-ins**.
+   >
+   > ![Screenshot showing Insert ribbon in Excel on Windows with the My Add-ins arrow highlighted.](../images/excel-cf-register-add-in-1b.png)
+   >
+   > In the list of available add-ins, find the **Developer Add-ins** section and select the your add-in to register it.
+
+   > [!TIP]
+   > The task pane may appear blank when it first opens. If so, it should render correctly when you launch the debugging tools in a later step.
+
+3. Open the [personality menu](../design/task-pane-add-ins.md#personality-menu) and then choose **Attach a debugger**. This will open the debugging tools for the webview control that Office is using to run add-ins on your Windows computer. You can set breakpoints and step through code as described in one of the following articles:
+
+    - [Debug add-ins using developer tools for Internet Explorer](../testing/debug-add-ins-using-f12-tools-ie.md)
+    - [Debug add-ins using developer tools for Edge Legacy](../testing/debug-add-ins-using-devtools-edge-legacy.md)
+    - [Debug add-ins using developer tools in Microsoft Edge (Chromium-based)](../testing/debug-add-ins-using-devtools-edge-chromium.md)
+
+4. To make changes to your code, first stop the debugging session in Visual Studio and close the Office application. Make your changes, and start a new debugging session.
+
+## Debug an Outlook add-in project
+
+This section describes how to start and debug an Outlook add-in.
+
+### Start the Outlook add-in project
+
+Start the project by choosing **Debug** > **Start Debugging** from the menu bar or press the F5 button. Visual Studio will automatically build the solution and launch the Outlook page of your Microsoft 365 tenancy.
 
 When Visual Studio builds the project it performs the following tasks.
 
-1. Creates a copy of the XML manifest file and adds it to  `_ProjectName_\bin\Debug\OfficeAppManifests` directory. The Office application that hosts your add-in consumes this copy when you start Visual Studio and debug the add-in.
+1. Prompts you for login credentials. If you're asked to sign in repeatedly or if you receive an error that you are unauthorized, then Basic Auth may be disabled for accounts on your Microsoft 365 tenant. In this case, try using a Microsoft account instead. You can also try setting the property **Use multi-factor auth** to **True** in the Outlook Web Add-in project properties dialog.
```

Comment:
Consider linking to the 'Add-in project properties' section (https://review.docs.microsoft.com/en-us/office/dev/add-ins/develop/debug-office-add-ins-in-visual-studio?branch=pr-en-us-3253#add-in-project-properties).